molecularnodes/ui/props.py

```diff
@@ -77,6 +80,110 @@ class EntityProperties(bpy.types.PropertyGroup):
     )  # type: ignore
 
 
+def _update_dssp_display_option(self, context):
+    entity = context.scene.MNSession.get(self.id_data.uuid)
+    if entity is None or self.cancelling:
+        return
+    if entity._entity_type == EntityType.MD_STREAMING:
+        display_option = getattr(self, "display_option_streaming")
+    else:
+        display_option = getattr(self, "display_option")
+    # call none and per-frame directly
+    if display_option == "none":
+        entity.dssp.show_none()
+        _update_entities(self, context)
+    elif display_option == "per-frame":
+        entity.dssp.show_per_frame()
+        _update_entities(self, context)
+    elif display_option == "sliding-window-average":
+        sw_threshold = self.sw_threshold if self.apply_sw_threshold else None
+        entity.dssp.show_sliding_window_average(self.window_size, sw_threshold)
+        _update_entities(self, context)
+    else:
+        self.applied = False
+
+
+def _update_dssp_applied(self, context):
+    if self.applied:
+        _update_entities(self, context)
+
+
+class DSSPProperties(bpy.types.PropertyGroup):
+    display_option: EnumProperty(  # type: ignore
+        name="Display",
+        description="Options to display secondary structures",
+        items=(
+            ("none", "None", "Do not show secondary structures"),
+            ("per-frame", "Per Frame", "Secondary structures calculated per frame"),
+            (
+                "sliding-window-average",
+                "Sliding Window Average",
+                "Average secondary structures of a sliding window of frames",
+            ),
+            (
+                "trajectory-average",
+                "Trajectory Average",
+                "Average secondary structures across all frames",
+            ),
+        ),
+        default="none",
```

Comment:
After doing some testing - I think the default should be `per-frame`. If the user is initializing the DSSP module then we should start with at least some value computed. The `none` option should still be there for someone to disable.